templates/CRM/Form/body.tpl

```diff
@@ -32,7 +32,7 @@
 {/if}
 
 {* Add all the form elements sent in by the hook  - formerly used by civiDiscount, now deprecated*}
-{if $beginHookFormElements}
+{if !empty($beginHookFormElements)}
```

Comment:
I was going to suggest that as per the code comment to maybe do something else, but the code comment isn't right anymore since the deprecation was reverted in https://github.com/civicrm/org.civicrm.module.cividiscount/pull/254. And also it looks like it's used in a small handful of other extensions.